docs/data/routes.json

```diff
@@ -5,6 +5,15 @@
         "breadTitle": "Home",
         "path": "/"
     },
+    "404": {
```

Comment:
I prefer to use "NotFound" as route name

Reply:
Done